portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,38 +28,42 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
- // public static void main(String[] args) {
-   // ArrayList<String> messages = new ArrayList<String>();
-  //  messages.add("How are you?");
-  //  messages.add("What did you have for breakfast?");
-  //  messages.add("How old are you?");
-  //  System.out.println(messages);
-//}
+  private ArrayList<String> messages = new ArrayList<String>();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String commentString = request.getParameter("data-input");
+      messages.add(commentString);
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     //response.setContentType("text/html;");
     //response.getWriter().println("<h1>Hello Nnenna!</h1>");
 
+    //TO DO: Remove commented code.
```

Comment:
Why not do it in this PR?

Reply:
I wanted to keep it incase I wanted to reference it in a later walkthrough, but I will delete it now because I am almost finished.

Reply:
I don't see the changes yet. Not sure if they are already done.

Reply:
Same. Could you update your remote repo